lib/manager/components/TransformationsViewer.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+
+import Icon from '@conveyal/woonerf/components/icon'
+import React, { Component } from 'react'
+import { Col, ListGroup, Panel, Row, ListGroupItem, Label } from 'react-bootstrap'
+
+import type { FeedVersion, TableTransformResult } from '../../types'
+
+type Props = {
+  version: FeedVersion,
+}
+
+export default class TransformationsViewer extends Component<Props> {
+  _getBadge (transformResult: TableTransformResult) {
+    switch (transformResult.transformType) {
+      // ESLint thinks that these are form labels
+      /* eslint jsx-a11y/label-has-for: "off" */
```

Comment:
You can `import { Label as BsLabel } from 'react-bootstrap` (and rename accordingly in `render`) to get rid of this linter exception.

Reply:
that would be better!